_posts/2016-05-05-clearing-the-air.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "Clearing the Air about Vulnerabilities in OpenStack"
+date:   2016-05-05
+categories: vulnerabilities
+tags: security vmt vulnerabilities summit
+author: "tmcpeak @ IBM Cloud, sicarie @ HPE"
+---
+
+Recently, there have been a few talks around "vulnerabilities" within
+the OpenStack project that missed the mark so much they deserve note.
```

Comment:
"missed the mark so much" probably isn't the right tone.  Maybe, "introduced some unfair concern"